install/kubernetes/helm/istio/values.yaml

```diff
@@ -142,7 +142,10 @@ ingressgateway:
     - containerPort: 80
     - containerPort: 443
     - containerPort: 31400
-    secretVolumes: []
+    secretVolumes:
+    - name: ingress-certs
+      secretName: istio-ingress-certs
+      mountPath: /etc/istio/ingress-certs
```

Comment:
I suggest adding the existing Istio CA volume as well as another volume called customcerts. Lets stay away from "istio-ingress", its going to confuse the heck out of users. 

Also please add same volumes to egress gateway as well.